JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -112,12 +146,12 @@ private fun BottomBar(post: Post, onUnimplementedAction: () -> Unit) {
 
 @Composable
 private fun BottomBarAction(
-    @DrawableRes id: Int,
-    onClick: () -> Unit
+        @DrawableRes id: Int,
+        onClick: () -> Unit
```

Comment:
nit: this might be related to codestyle missconfig in AS, you can also run ktlint to format it automatically